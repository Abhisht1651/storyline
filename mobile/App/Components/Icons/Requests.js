import React from 'react'
import { View, Text } from 'react-native'

// Styles
import styles from '../Styles/IconStyles/RequestsStyles'
import RequestsSVG from '../../Images/Icons/icon-friend-requests.svg'

const Requests = () => {
  return (
    <View style={styles.cardBox}>
      <View style={styles.profileIconBox}>
        <RequestsSVG />
      </View>
      <View style={styles.TextBox}>
        <Text style={styles.TextStyles}>Friend Requests</Text>
      </View>
    </View>
  )
}

export default Requests
