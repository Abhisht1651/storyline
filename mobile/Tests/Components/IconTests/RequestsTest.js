import React from 'react'
import 'react-native'
import renderer from 'react-test-renderer'

import Requests from '../../../App/Components/Icons/Requests'

test('Requests Icon component renders correctly', () => {
  const tree = renderer.create(<Requests />).toJSON()
  expect(tree).toMatchSnapshot()
})
