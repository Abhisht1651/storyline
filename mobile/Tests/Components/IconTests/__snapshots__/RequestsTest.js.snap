// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Requests Icon component renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "transparent",
      "flexDirection": "row",
      "height": 25,
      "opacity": 1,
      "width": 180,
    }
  }
>
  <View
    style={
      Object {
        "height": 25,
        "width": 25,
      }
    }
  >
    <SvgMock />
  </View>
  <View
    style={
      Object {
        "height": 20,
        "marginLeft": 13,
        "marginTop": 3,
        "width": 144,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#1F1F60",
          "fontFamily": "BananaGrotesk-Medium",
          "fontSize": 18,
          "letterSpacing": -0.12,
          "opacity": 1,
          "textAlign": "left",
        }
      }
    >
      Friend Requests
    </Text>
  </View>
</View>
`;
